charts/catalog/templates/rbac.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
```

Comment:
@MHBauer I believe we should be able to use the following to test whether the rbac API group is available:

```yaml
{{- if .Capabilities.APIVersions.Has "rbac.k8s.io/v1beta1" }}
// stuff
{{- end }}
```